src/main.cc

```diff
@@ -644,6 +644,53 @@ Napi::String _getWindowTitle(const Napi::CallbackInfo &info) {
     return Napi::String::New(env, getWindowTitle(windowHandle));
 }
 
+Napi::Boolean _focusWindow(const Napi::CallbackInfo &info) {
+    Napi::Env env = info.Env();
+
+    WindowHandle windowHandle = info[0].As<Napi::Number>().Int64Value();
+    
+    bool result = focusWindow(windowHandle);
+    
+    return Napi::Boolean::New(env, result);
+}
+
+Napi::Boolean _resizeWindow(const Napi::CallbackInfo& info) {
+    Napi::Env env = info.Env();
+
+    if (info.Length() < 2 || !info[0].IsNumber() || !info[1].IsObject()) {
+        Napi::TypeError::New(env, "Invalid arguments. Expected handle (number) and rect (object).").ThrowAsJavaScriptException();
+        return Napi::Boolean::New(env, false);
+    }
+
+    WindowHandle windowHandle = info[0].As<Napi::Number>().Int64Value();
+    MMRect windowRect = getWindowRect(windowHandle);
+    Napi::Object rectObj = info[1].As<Napi::Object>();
+
+    int64_t x = 0;
+    int64_t y = 0;
+    int64_t width = 0;
+    int64_t height = 0;
+
+    if (rectObj.Has("x"))
+        x = rectObj.Get("x").As<Napi::Number>().Int64Value();
+    if (rectObj.Has("y"))
+        y = rectObj.Get("y").As<Napi::Number>().Int64Value();
+    if (rectObj.Has("width"))
+        width = rectObj.Get("width").As<Napi::Number>().Int64Value();
+    if (rectObj.Has("height"))
+        height = rectObj.Get("height").As<Napi::Number>().Int64Value();
+
+    windowRect.origin.x = x;
+    windowRect.origin.y = y;
+    windowRect.size.width = width;
+    windowRect.size.height = height;
+
+    resizeWindow(windowHandle, windowRect);
+
+    return Napi::Boolean::New(env, true);
```

Comment:
**Suggestion**: `resizeWindow` returns a boolean, so we should return this result here.